demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -147,6 +147,11 @@ def test_get_remote_file_sanity(self):
         assert hello_world_yml
         assert hello_world_yml['commonfields']['id'] == 'HelloWorld'
 
+    def test_get_remote_file_content_sanity(self):
```

Comment:
can we add 1 more test case(non sanity) with an example of how the `file.content` looks like?